.github/policies/labelAdded.highestVersionRemoval.yml

```diff
@@ -0,0 +1,36 @@
+id: labelAdded.highestVersionRemoval
+name: GitOps.PullRequestIssueManagement
+description: Handlers when "Highest-Version-Removal" label is added
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+      - description: >-
+          When the label "Highest-Version-Removal" is added to a pull request
+          * Add a reply notifying the issue author
+          * Assign to the author
+          * Label with Needs-Author-Feedback
+        if:
+          - payloadType: Pull_Request
+          - labelAdded:
+              label: Highest-Version-Removal
+        then:
+          - addReply:
+              reply: >-
+                Hello @${issueAuthor},
+
+                This PR removes the highest available version of the package from the repository.
+
+                Please provide the reason for removal to help our moderators understand the context.
```

Comment:
You need the empty lines for the proper formatting

Reply:
Do you mean the line below this? These empty lines seem present. Feel free to suggest the changes you want to see.